docs/src/main/paradox/reference.md

```diff
@@ -0,0 +1,64 @@
+# Reference
+
+This is the reference documentation for an Alpakka connector. This section should contain
+a general overview of the connector and mention the libraries and APIs that the connector
+is using. Also it should link to external resources that might help to learn
+about the technology connector is using.
+
+### Reported issues
+
+[Tagged issues at Github](https://github.com/akka/alpakka/labels/p%3Areference)
+
+## Artifacts
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-stream-alpakka-reference_$scalaBinaryVersion$
+  version=$version$
+}
+
+## Usage
+
+Give a brief description of the usage of this connector. If you want to mention a
+class name, make sure to link to the API docs:
+@scaladoc[ReferenceReadMessage](akka.stream.alpakka.reference.ReferenceReadMessage).
+
+### Reading messages
```

Comment:
Make the interesting sections on level 2 to let them show in the page menu.
Possibly removing the Usage section.